src/ppx/application.cpp

```diff
@@ -1225,12 +1270,26 @@ int Application::Run(int argc, char** argv)
         PPX_LOG_INFO("Display not enabled: frame count is " + std::to_string(mMaxFrame));
     }
 
+#if defined(PPX_BUILD_XR)
+    if (mSettings.enableXR) {
+        mXrComponent.InitializeBeforeGrfxDeviceInit(mSettings.grfx.enableDebug);
+        // TODO(wangra): disable ImGui for XR for now, need to be fixed
```

Comment:
Feel free to reference b/ bugs instead of using usernames. It's explicitly allowed even for open source projects. In this case, it's b/256679355.

Reply:
fixed